scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -0,0 +1,70 @@
+
+Telco edge computing presents extraordinary challenges with managing hundreds to tens of thousands of clusters in hundreds of thousands of locations. These challenges require fully-automated management solutions with, as closely as possible, zero human interaction.
+
+Zero touch provisioning (ZTP) addresses the problem of provisioning new edge sites from the moment there is network connectivity to the site and until the CNF workload is deployed and running on the site nodes.
+
+ZTP eliminates the need for service providers to manually configure bare-metal equipment at remote sites. Manual intervention is eliminated. Declarative frameworks of desired end states are pushed out into the environment. Template or overlay configurations install {product-title} features that are required for CNF workloads and an end-to-end functional test suite to verify CNF related features. All Kustomize configs are entirely declarative in nature.
+
+Customers start by creating and refining ISO images of the node configuration they want. A unique image for each node is created based on the configuration provided. The images are then used to repeatedly provision large numbers of nodes efficiently and quickly, allowing the customer to keep up with the needs from the field for far edge nodes.
+
+Service providers are deploying a more distributed mobile network architecture allowed by the modular functional framework defined for 5G. This allows service providers to move from appliance-based radio access networks (RAN) to open cloud RAN architecture, gaining flexibility and agility in delivering services to end users.
+
+include::modules/ztp-ztp-building-blocks.adoc[leveloffset=+1]
+
+include::modules/ztp-about-ztp-and-distributed-units-on-single-node-openshift-clusters.adoc[leveloffset=+1]
+
+include::modules/ztp-low-latency-for-distributed-units-dus.adoc[leveloffset=+1]
+
+include::modules/ztp-single-node-clusters.adoc[leveloffset=+1]
+
+include::modules/ztp-the-gitops-approach.adoc[leveloffset=+1]
+
+//planning and prereqs
+
+include::modules/ztp-acm-preparing-to-install-disconnected-acm.adoc[leveloffset=+1]
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+Deploying a disconnected registry host based on the `docker.io/library/registry:2` API for {product-title} is not officially supported by Red Hat. You can create a mirror host based on the `docker.io/library/registry:2` API with the following unsupported procedure.
+====
+
+include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+You must have access to the internet to obtain the necessary container images.
+In this procedure, you place the mirror registry on a mirror host
+that has access to both your network and the Internet. If you do not have access
+to a mirror host, use the disconnected procedure to copy images to a device you
+can move across network boundaries with.
+====
+
+[id="installing-preparing-mirror_{context}"]
```

Comment:
I think this heading is too late in the assembly. I think that the two modules you have about the mirror registry might be more effective under this heading, and I strongly suggest moving at least the procedure under this heading.

The notes that surround `include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]` aren't rendering in the most helpful spots - one is absorbed into the concept and is easy to skip, and one comes after the procedure and is pretty useless at that point. I suggest either editing the module to move the notes into it (there's a similar note in the procedure module that you can modify the if statements to make show up here, if you want) or to move the notes under this subheading along with the procedure module.

Reply:
I think I've addressed this. hard to see the wood from the trees sometimes.